test_data/merge_conflict/source/index.txt

```diff
@@ -0,0 +1,140 @@
+================================
+Deploy a Cluster through the API
+================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. _create-cluster-with-api-overview:
+
+Overview
+--------
+
+This tutorial manipulates the :doc:`Public API's </core/api>`
+automation configuration to deploy a :term:`sharded cluster` that is
+owned by another user. The tutorial first creates a new project, then a
+new user as owner of the project, and then a sharded cluster owned by
+the new user. You can create a script to automate these procedures for
+use in routine operations.
+
+.. cond:: cloud
+
+   To perform these steps, you must have sufficient access to |mms|. A
+   user with the :authrole:`Project Owner` role has sufficient access.
+
+.. cond:: onprem
+
+   To perform these steps, you must have sufficient access to |mms|. A
+   user with the :authrole:`Global Owner` or
+   :authrole:`Project Owner` role has sufficient access.
+
+The procedures install a cluster with two :term:`shards <shard>`. Each
+shard comprises a three-member :term:`replica set`. The tutorial
+installs one |mongos| and three
+:term:`config servers <config server>`. Each component of the cluster
+resides on its own server, requiring a total of 10 servers.
+
+The tutorial installs the
+:doc:`{+mdbagent+} </tutorial/nav/install-mongodb-agent>` on each
+host.
+
+.. _create-cluster-with-api-prerequisites:
+
+Prerequisites
+-------------
+
+.. cond:: onprem
+
+   |mms| must have an existing user. If you are deploying the sharded
+   cluster on a fresh install of |mms|, you must register the first
+   user.
+
+   You must have the URL of the |onprem| Web Server, as set in the
+   You must have the |url| of the |onprem| Web Server, as set in the
+   ``mmsBaseUrl`` setting of the :doc:`{+magent+} configuration
+   file </reference/monitoring-agent>`.
+
+   Provision ten servers to host the components of the :term:`sharded
+   cluster`. For server requirements, see the :manual:`Production Notes
+   </administration/production-notes>` in the MongoDB manual.
+
+<<<<<<< HEAD
```

Comment:
It might be useful to have another sample conflict where the before or after is empty